### ISSUE TITLE
[Extension]: Flickering and Noisy Live Preview Images with Agent Scheduler

### ISSUE THREAD

**Issue by MDMAchine** [vladmandic/automatic#1717]
### Issue Description

Hope I did this correct this time (don't want to waste anyones time!) :)

When utilizing the agent scheduler, I have observed erratic behavior in the live preview images:

a) Flickering: The live preview images flicker intermittently during the diffusion process.
b) Random Noise: Random noise patterns appear in the live preview images, disrupting their visual quality.
c) General Odd Behavior: The live preview images display unexpected behavior, which may include artifacts, unusual color shifts, or distorted shapes.

Although these issues affect the live preview images, the final output images seem to be unaffected. It happens while its processing and/or if something is queued.

I am using --medvram

### Version Platform Description

Browser: Brave (MacOS)
app: "SD.next",
updated: "2023-07-15",
hash: "a7f5e75a",
torch: "2.0.1+cu118 Autocast half",
device: "NVIDIA GeForce RTX 3060 Ti (1) (compute_37) (8, 6)",
cuda: "11.8",
cudnn: 8700,
driver: "531.79"
optimizations: "medvram"
xformers: "0.0.20",
diffusers: "0.18.2",
transformers: "4.30.2"
cpu: "AMD64 Family 25 Model 80 Stepping 0, AuthenticAMD",
system: "Windows",
release: "Windows-10-10.0.22621-SP0",
python: "3.10.6"
crossattention: "xformers",
backend: "cuda",
pipeline: "original"

### URL link of the extension

https://github.com/ArtVentureX/sd-webui-agent-scheduler

### URL link of the issue reported in the extension repository

https://github.com/ArtVentureX/sd-webui-agent-scheduler/issues/80

### Acknowledgements

- [X] I have read the above and searched for existing issues

**Comment by slimeqslimedog** [comment]
I've found this to occur as well, on a 2080Ti, although I've had it occur in the main screen (i.e. not using the agent scheduler). Also, after a number of image generations have passed (usually a few dozen), I'll end up with either a CUDA invalid memory access or just a total lockup of whatever thread does image generation. At that point I have to kill and restart SD.Next to get things working again. The corruption and lockup/crash would occur faster if "second pass" or "ADetailer" is enabled.

*However*, after I changed the live preview mode from "Full VAE" to "Approximate NN", this seems to have stopped happening -- I can generate hundreds of images without any problems. I'm no CUDA or Python expert, but this suggests to me that there may be a GPU memory or other resource leak somewhere in the Live Previews code.
